src/rgw/rgw_kms.cc

```diff
@@ -966,19 +968,20 @@ int reconstitute_actual_key_from_kms(CephContext *cct,
 
   if (RGW_SSE_KMS_BACKEND_TESTING == kms_backend) {
     std::string key_selector = get_str_attribute(attrs, RGW_ATTR_CRYPT_KEYSEL);
-    return get_actual_key_from_conf(cct, key_id, key_selector, actual_key);
+    return get_actual_key_from_conf(dpp, cct, key_id, key_selector, actual_key);
   }
 
   ldout(cct, 0) << "ERROR: Invalid rgw_crypt_s3_kms_backend: " << kms_backend << dendl;
```

Comment:
please change this to ldpp_dout

Reply:
Do you mean that specific ldout call? because there are many more ldout calls in that file that I didn't change because I replaced only the calls in rgw_crypt.cc/h files. 
the changes in rgw_kms are for the calls to functions in rgw_crypt.cc.

Reply:
No, I mean if you're adding dpp to a function in some files other than rgw_crypt.cc/h then there you can change ldouts/lderrs->ldpp_dout in the same function (if any).